packages/app/backstop.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Is this file used?

Reply:
Double checked and its not. Removed it now. Same with the backstop js in ember-backstop.